SuperDARN/rst
# Bug fixes in time_plot

This PR fixes a few small bugs in `time_plot` introduced in #235  and #209 

To test: 

1. Run `time_plot -x -a -old [oldfitfile]` and check that the data displays correctly
2. Check that you are able to specify color keys for the lag zero power and phase parameters
e.g. `time_plot -x -p0 -p0key grey.key -phi -phikey green.key [fitacffile]`
